### ISSUE TITLE
Start up error: Using libsoup2 and libsoup3 in the same process is not supported

### ISSUE THREAD

**Issue by sommio** [johnfactotum/foliate#816]
```
Gjs-Message: 18:11:53.498: JS WARNING: [resource:///com/github/johnfactotum/Foliate/js/main.js 23]: Requiring WebKit2 but it has 2 versions available; use imports.gi.versions to pick one

(com.github.johnfactotum.Foliate:1644): libsoup-ERROR **: 18:11:53.616: libsoup3 symbols detected. Using libsoup2 and libsoup3 in the same process is not supported.
fish: Job 1, 'foliate' terminated by signal SIGTRAP (Trace or breakpoint trap)
```

**To Reproduce**
Run foliate

**Version:**
 - Foliate version: 2.6.3-1
 - OS/Distribution and version: Arch Linux
 - Desktop environment: GNOME 41.1 (Xorg)
 - Installation method: Pacman

**Comment by johnfactotum** [comment]
Probably a duplicate of #812 and should be fixed with #808

**Comment by gooycity** [comment]
After I uninstall the tracker and trancker miners, the foliate can be opened.

My environment is to minimize the installation of Gnome.
